tests/test_sbr_calculator.py

```diff
@@ -0,0 +1,353 @@
+from src.smart_building_rating_calculator.calculate_sbr_score import (
+    get_sbr_scores,
+    sbr_score,
+)
+from src.smart_building_rating_calculator.flex_archetype import FlexArchetype
+from src.smart_building_rating_calculator.inputs import (
+    BatterySize,
+    EVChargerPower,
+    HeatingSource,
+    HotWaterSource,
+    SolarInverterSize,
+    UserInputs,
+)
+from src.smart_building_rating_calculator.intermediate_scoring import (
+    calc_electrification_score,
+    calc_ics_score,
+    calc_smart_meter_score,
+)
+
+
+class TestScoreCalculators:
+    """Test the scoring calculations using the results in the SBR excel sheet created by ESC"""
+
+    def test_smartest_home(self):
+        user_inputs = UserInputs(
+            smart_meter=True,
+            smart_ev_charger=True,
+            charger_power=EVChargerPower.CHARGER_7KW,
+            smart_v2g_enabled=True,
+            home_battery=True,
+            battery_size=BatterySize.LARGE,
+            solar_pv=True,
+            pv_inverter_size=SolarInverterSize.LT_4KW,
+            electric_heating=True,
+            heating_source=HeatingSource.HEAT_PUMP,
+            hot_water_source=HotWaterSource.HEAT_BATTERY_OR_ELEC_HOT_WATER_TANK,
+            secondary_heating=True,
+            secondary_hot_water=True,
+            integrated_control_sys=True,
+        )
+        elec_scores = calc_electrification_score(user_inputs)
```

Comment:
You should be unit testing these functions

Reply:
Let me know if better now please. Thanks!